CHANGELOG.md

```diff
@@ -6,6 +6,15 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [10.1.0] - 2021-06-7
+### Added
+- Export BaseControllerV2 Json type ([#482](https://github.com/MetaMask/controllers/pull/482))
+
+### Changed
+- Improve collectible detection by account ([#487](https://github.com/MetaMask/controllers/pull/487))
+- Mapping token address strings instead of Token objects while checking for new tokens ([#480](https://github.com/MetaMask/controllers/pull/480))
```

Comment:
This PR was more of a bug fix, or at least that was my understanding. I would put this under Fixed, with the description "Skip token detection for tokens that are already tracked", or something like that. That was the bug - that we're detecting _all_ tokens, even those already on the user's list.